src/actor.js

```diff
@@ -485,6 +487,68 @@ export const callTask = async (taskId, input, options = {}) => {
     return Object.assign({}, updatedRun, { output });
 };
 
+
+/**
+ * Metamorphs current actor run into a run of another actor with given input.
+ * After this function is called the actor container gets stopped and restarted as a run of given actor.
```

Comment:
I think this needs a much better explanation in the docs. 2 sentences just don't cut it. I don't even know how I would use it.

Reply:
Yeah good point. Also, I wouldn't use the word "metamorph" in the explanation, since that's exactly the thing we need to explain :)